testing/dbtest/pgtap-docker/Makefile

```
@@ -0,0 +1,20 @@
+REGISTRY_NAME?=docker.io/hashicorpboundary
+TEST_IMAGE_NAME=pgtap
+PG_VERSION=13
```

```suggestion
PG_VERSION ?= 13
```

This will allow this to be set easily, ie `PG_VERSION=11 make docker-build`